test/Installer/testmsi.ps1

```diff
@@ -42,7 +42,6 @@ pushd "$Stage2Dir\bin"
 
 try {
     .\dotnet restore `
-        --runtime win-anycpu `
```

Comment:
/cc @Sridhar-MS as far as I could see, there weren't any runtime-specific assets to warrant using `--runtime`... was this added for a specific reason? `win-anycpu` isn't even actually a valid RID :)